File: Web_Api/Rest_NetApi.Domain/Entities/Station.cs
using Rest_NetApi.Domain.DBObjects;
using Rest_NetApi.Domain.DTOs;
using Rest_NetApi.Domain.Interface.IEntity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Rest_NetApi.Domain.Entities
{
    public class Station : IStation
    {
        public Guid Id { get ; set ; }
        public string name { get ; set ; }
        public string address { get ; set ; }
        public double latitutde { get ; set ; }
        public double longitude { get ; set ; }
        public int capacity { get ; set ; }
        public int freeDocks { get ; set ; }
        public int totalGets { get ; set ; }
        public int totalReturns { get ; set ; }
        public int availableBikeShared { get ; set ; }

        public Station() { }
        public Station(IStation station_): this(station_.Id, station_.name, station_.address, station_.latitutde, station_.longitude, station_.capacity, station_.freeDocks, station_.totalGets, station_.totalReturns, station_.availableBikeShared) { }
        public Station(Guid id, string name, string address, double latitutde, double longitude, int capacity, int freeDocks, int totalGets, int totalReturns, int availableBikeShared)
        {
            Id = id;
            this.name = name;
            this.address = address;
            this.latitutde = latitutde;
            this.longitude = longitude;
            this.capacity = capacity;
            this.freeDocks = freeDocks;
            this.totalGets = totalGets;
            this.totalReturns = totalReturns;
            this.availableBikeShared = availableBikeShared;
        }
        public StationDto ToDto()
        {
            return new StationDto(this.Id, this.name, this.address, this.latitutde, this.longitude, this.capacity, this.freeDocks, this.totalGets, this.totalReturns, this.availableBikeShared);
        }

    }
}

File: Android_App/lib/models/station.dart
import '../Helpers/Parse.dart';
class Station{
  String stationId;
  String name;
  String address;
  double lat;
  double long;
  int capacity;
  int freeDocks;
  int totalGets;
  int totalReturns;
  int availableBikeShared;

  Station({
    required this.stationId,
    required this.name,
    required this.address,
    required this.lat,
    required this.long,
    required this.capacity,
    required this.freeDocks,
    required this.totalGets,
    required this.totalReturns,
    required this.availableBikeShared,
  });

  Station.fromJson(Map<String, dynamic> json)
      : stationId= json['id'] as String,
        name = json['name'] as String,
        address = json['address'] as String,
        lat = Helping.checkDouble(json['latitutde']),// as double,
        long =Helping.checkDouble(json['longitude']),// as double, //Helping.checkDouble(json['credit']),
        capacity = json['capacity'] as int,
        freeDocks = json['freeDocks'] as int,
        totalGets= json['totalGets'] as int,
        totalReturns = json['totalReturns'] as int,
        availableBikeShared = json['availableBikeShared'] as int;

}